time/Cargo.toml

```
@@ -26,7 +26,7 @@ default = ["std"]
 alloc = ["serde?/alloc"]
 formatting = ["dep:itoa", "std", "time-macros?/formatting"]
 large-dates = ["time-macros?/large-dates"]
-local-offset = ["std"]
+local-offset = ["std", "libc", "num_threads"]
```

```suggestion
local-offset = ["std", "dep:libc", "dep:num_threads"]
```

This avoids introducing implicit feature flags.

---

Yeah there's so much stuff you can do that I don't know about, the dep thing is nice though, should have spotted it from the other imports!

---

No worries! It's relatively new, only about 7 months old on stable. I did add a note (mostly for myself) above `[dependencies]`, but you had no reason to look there.